### PR TITLE
added fixture and skipifs to the community tests

### PR DESCRIPTION
**Pull request by 3rd-Son**
I added `fixture` and `skipifs` to the necessary test cases in `swarmauri_community`
I also updated the `setup.py` in the `swarmauri_community` 

currently, below is the log summary of the test
```
===================================================================================================== short test summary info =====================================================================================================
FAILED tests/unit/tools/EntityRecognitionTool_test.py::test_call - RuntimeError: Failed to import transformers.models.bert.modeling_tf_bert because of the following error (look up to see its traceback):
FAILED tests/unit/tools/PsutilTool_test.py::test_call[network-expected_keys3-False] - psutil.AccessDenied: (pid=11947)
FAILED tests/unit/tools/PsutilTool_test.py::test_call[sensors-expected_keys4-False] - AttributeError: module 'psutil' has no attribute 'sensors_temperatures'
FAILED tests/unit/tools/SentimentAnalysisTool_test.py::test_call[I love this product!-expected_labels0] - RuntimeError: Sentiment analysis failed: Failed to import transformers.models.distilbert.modeling_tf_distilbert because of the following error (look up to see its traceback):
FAILED tests/unit/tools/SentimentAnalysisTool_test.py::test_call[I hate this product!-expected_labels1] - RuntimeError: Sentiment analysis failed: Failed to import transformers.models.distilbert.modeling_tf_distilbert because of the following error (look up to see its traceback):
FAILED tests/unit/tools/SentimentAnalysisTool_test.py::test_call[This product is okay.-expected_labels2] - RuntimeError: Sentiment analysis failed: Failed to import transformers.models.distilbert.modeling_tf_distilbert because of the following error (look up to see its traceback):
ERROR tests/unit/vector_stores/Neo4jVectorStore_test.py::test_ubc_type - neo4j.exceptions.ServiceUnavailable: Couldn't connect to localhost:7687 (resolved to ()):
ERROR tests/unit/vector_stores/Neo4jVectorStore_test.py::test_add_document - neo4j.exceptions.ServiceUnavailable: Couldn't connect to localhost:7687 (resolved to ()):
ERROR tests/unit/vector_stores/Neo4jVectorStore_test.py::test_add_documents - neo4j.exceptions.ServiceUnavailable: Couldn't connect to localhost:7687 (resolved to ()):
================================================================================ 6 failed, 113 passed, 145 skipped, 6 warnings, 3 errors in 40.46s ================================================================================
```

What do you have to say about the errors here

### CODE REVIEW

File: pkgs/community/tests/unit/vector_stores/Neo4jVectorStore_test.py
Comment (MichaelDecent) [inline pkgs/community/tests/unit/vector_stores/Neo4jVectorStore_test.py:1]: Can we remove the defaults values when fetching the `env` variables, create the actual env variables, and try using them, to see if we will reproduce same error. `NEO4J_PASSWORD` and
`NEO4J_URI`

Comment (MichaelDecent) [reply pkgs/community/tests/unit/vector_stores/Neo4jVectorStore_test.py:1]: i will DM mine to test with

Comment (3rdsonx) [reply pkgs/community/tests/unit/vector_stores/Neo4jVectorStore_test.py:1]: okay
Waiting for the DM

Comment (MichaelDecent) [reply pkgs/community/tests/unit/vector_stores/Neo4jVectorStore_test.py:1]: sent already

**Review by cobycloud — APPROVED** [review]

**Pull request merged**